### PR TITLE
Latest gradle wrapper (2.12) and android gradle plugin (1.5.0)

### PR DESCRIPTION
**Pull request by wmathurin**

### CODE REVIEW

File: gradle/wrapper/gradle-wrapper.properties [hunk 1]
```diff
@@ -1,6 +1,6 @@
-#Mon Dec 08 15:01:36 PST 2014
+#Mon Apr 04 13:28:13 PDT 2016
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-2.3-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-2.12-bin.zip
```
Comment (bhariharan) [inline gradle/wrapper/gradle-wrapper.properties]: ```2.12``` is more recent than ```2.3```? Weird numbering scheme.

**Comment by bhariharan** [comment]
:+1: 

Any side effects?

**Comment by wmathurin** [comment]
No issues building or running tests. I'll try the sample apps next.

**Comment by bhariharan** [comment]
Sample apps will tell us if ```Instant Run``` works. That'll be pretty sweet!

**Pull request merged**